staging/src/k8s.io/apiserver/pkg/registry/rest/delete.go

```diff
@@ -39,6 +39,7 @@ type GarbageCollectionPolicy string
 const (
 	DeleteDependents GarbageCollectionPolicy = "DeleteDependents"
 	OrphanDependents GarbageCollectionPolicy = "OrphanDependents"
+	Unsupported      GarbageCollectionPolicy = "Unsupported"
```

Comment:
> I see. All "unsupported" resources should also be in the "ignored" list of GC. Could you say that explicitly in the comment of Unsupported? The system will behave strangely if the "two sets of ignores" don't match.

I can, but do you think this should be exposed via discovery?  I think there's a good argument to do that.